Move mailerlite resources internally to fix CORP errors

danyeaw opened the pull request:
# Internal Hosting of MailerLite Assets

## Problem
- Issue #80 reports errors caused by MailerLite's incorrect Cross-Origin Resource Sharing (CORS) configuration
- External CSS and JavaScript files from MailerLite were failing to load properly
- Discussed issue with @WebReflection

## Solution
- Moved from using MailerLite's files, made updates to the html, CSS, and JavaScript that is now hosted internally in our codebase
- Eliminates dependency on MailerLite's CORS configuration

This change should resolve the resource loading errors and restore full MailerLite functionality.

## Other Chores
- I also minified the js and css files to improve page load speed slightly
- Bumped the PyScript version to the latest

## Style
While making the change, I also updated the styling to better fit with the rest of the site:

### Before:
<img width="783" alt="image" src="https://github.com/user-attachments/assets/43fe7988-4af9-448a-8cb4-313dc7396ee2" />

### After:
<img width="793" alt="image" src="https://github.com/user-attachments/assets/d633025b-c4c6-4da8-921e-865365fbb7fb" />
<img width="793" alt="image" src="https://github.com/user-attachments/assets/cfb6e843-d115-4baf-b923-04c4ef6d52d1" />

WebReflection commented on index.html:
```diff
@@ -13,13 +13,15 @@
-		<link rel="stylesheet" href="assets/css/main.css?v=1.5">
+    <link rel="stylesheet" href="/assets/css/main.min.css?v=1.6">
+
+    <script src="assets/js/mailerlite-script.min.js?=v1.0" defer></script>
```
nit: any particular reason this asset is not absolute like the css one, or vice-versa?

danyeaw replied:
Thanks @WebReflection, I pushed a change to make them all absolute.

WebReflection reviewed (approved):
beside the *nit* (not so important, rather curious) I see the `index.html` lost a lof of CSS code, including code that looks produced by TailwindCSS or similar tools, such as `#mlb2-17564783.ml-form-embedContainer .ml-form-embedWrapper .ml-form-embedBody` and similar ... if you can confirm that is intentional and everything looks fine with these changes in (appreciated, 'cause the cleanup is quite huge) feel free to merge, thanks.

danyeaw commented:
Hi @WebReflection, the `mlb2-...` stuff was copy and pasted from MailerLite's setup instructions. I renamed it to newsletter when I moved it internally to make the CSS more clear and manageable.

WebReflection commented:
@danyeaw awesome then! feel free to merge or maybe make the *JS* also absolute with a starting `/` path and then merge, either works with the current setup anyway, I just found it odd CSS is absolute, JS is not but also, here it doesn't matter, it would if we move that folder in a subfolder, if ever.

danyeaw commented:
@WebReflection I do not have merge permissions :)
